proto/Cargo.toml

```diff
@@ -15,8 +15,12 @@ description = """
 
 [features]
 default = []
+std = []
```

Comment:
I'm getting a bunch of build failures trying to use this feature:

```
$ cargo test --all-features                                                                                       101 ↵
   Compiling tendermint-proto v0.39.0
error[E0433]: failed to resolve: use of undeclared crate or module `std`
   --> /Users/tarcieri/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tendermint-proto-0.39.0/src/google/protobuf/timestamp.rs:105:11
```

I think you forgot an `extern crate std` and it seems like CI probably isn't testing it.

Reply:
Opened an issue to track this: #1454